src/App.jsx

```diff
@@ -14,33 +15,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const NOT_SORTED = '';
+const SORTED_BY_NAME = 'name';
+const SORTED_BY_LENGTH = 'length';
+const REVERSED = 'reversed';
+const NOT_REVERSED = 'not reversed';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goodsFrom, sort, revStatus) {
```

Comment:
You can move this function into a separate file. 
Then import and use as a utility or helper function.